### DISCUSSION TITLE
Stream EC2 (Hubs Cloud on AWS)  TLS settings

### DISCUSSION THREAD

**Discussion by ks-aa** [mozilla/hubs#4123] [Hubs Cloud Questions] [answered]
I conducted a vulnerability check on Hubs Cloud (Enterprise) running on AWS.
As a result, a vulnerability related to SSL settings was discovered for EC2 for Stream.
It is necessary to update the TLS version to address the vulnerability. Which setting should be changed?

**Comment by pattersonbl2** [comment, marked as answer]
I have opened a GitHub issue for this.  Do you have anymore information on this vulnerability ?   if so please add to this GitHub issue. (https://github.com/mozilla/hubs/issues/4571)
